pep-0675.rst

```diff
@@ -747,13 +737,67 @@ others may consider it in the future, such extensions are out of scope
 for this PEP.
 
 
+Rejected Names
+--------------
+
+We considered a variety of names for the literal string type and
+solicited ideas on `typing-sig
+<https://mail.python.org/archives/list/typing-sig@python.org/thread/VB74EHNM4RODDFM64NEEEBJQVAUAWIAW/>`_. Some
+notable alternatives were:
+
++ ``Literal[str]``: This is a natural extension of the
+  ``Literal["foo"]`` type name but typing-sig `objected
+  <https://mail.python.org/archives/list/typing-sig@python.org/message/2ZQO4NTJEI42KTRJDBL77MNANEXOW7UI/>`_
+  that users could mistake this for the literal type of the ``str``
+  class.
+
++ ``LiteralStr``: This is shorter than ``LiteralString`` but looks
```

Comment:
```suggestion
+ ``LiteralStr``: This is shorter than ``LiteralString``, but looks
```

Missing comma

Reply:
No, it's not an independent clause.

Reply:
Looks like this was not applied.

Reply:
Yes, and I explained why I did not apply it: "looks weird to the authors" is not an independent clause. We don't add a comma before "but" in those cases.

Reply:
It wasn't syntactically required, but it sounded better to me as a copyeditor in terms of how I read it. Reading it again, though, and getting a second opinion from my sister (a professional writer, whom I sometimes copyedit for), I agree you're right that it sounds fine without it.